src/layouts/core/coreLayoutSSG.ts

```diff
@@ -109,53 +109,16 @@ export const getCoreStaticProps: GetStaticProps<SSGPageProps, CommonServerSidePa
   const bestCountryCodes: string[] = [lang, resolveFallbackLanguage(lang)];
   const gcmsLocales: string = prepareGraphCMSLocaleHeader(bestCountryCodes);
   const i18nTranslations: I18nextResources = await getLocizeTranslations(lang);
-  // XXX This part is not using "getGraphcmsDataset" because I'm not sure how to return the "apolloClient" instance when doing so, as it'll be wrapped and isn't returned
-  //  So, code is duplicated, but that works fine
+  const dataset: StaticDataset | GraphCMSDataset = await getGraphcmsDataset(gcmsLocales);
   const apolloClient: ApolloClient<NormalizedCacheObject> = initializeApollo();
```

Comment:
I'm not sure if `apolloClient` will actually contain the cached data in such case, and that's why I haven't de-dupe the code.

Reply:
Since the data is being statically cached, what's the point of Apollo cache?

Reply:
AFAIU, if apollo is loaded in the browser without the cache, then it'll need to re-fetch during the React reconciliation phase. I believe the cache is what's make the data available from the backend to the frontend, but I might be wrong, or it might be possible to do it another way. I'm not so experienced with the Apollo's internals so I preferred to duplicate code rather than create a bug/regression.

But, as you stated, since the data are statically available, it makes sense there would be a way to write the code only once.